doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -135,6 +135,11 @@ RF Front-End Modules
   The :ref:`MPSL library <nrfxlib:mpsl>` is linked into the build without initialization.
   You cannot use other MPSL features when this option is enabled.
 
+Secure Partition Manager (SPM)
+==============================
+
+* The Secure Partition Manager (SPM), kconfig option :kconfig:option:`CONFIG_SPM`, is now deprecated. It is replaced by Trusted Firmware-M (TF-M) as the supported solution. For information about TF-M see :ref:`TF-M <ug_tfm>`.
```

Comment:
```suggestion
* The Secure Partition Manager (SPM), the Kconfig option :kconfig:option:`CONFIG_SPM`, is now deprecated.
  It is replaced by Trusted Firmware-M (TF-M) as the supported solution.
  For information about TF-M see :ref:`TF-M <ug_tfm>`.
```

Reply:
@mia-ko  please approve when you verify the changes.